tools/policy_sync/CHANGELOG.md

```diff
@@ -1,48 +1,52 @@
-# CHANGELOG
-
-## 1.10
-
-- fix an issue with publishing
-
-## v1.9
-
-- use the info field for version.
-
-## v1.8
-
-- update github org in path
-
-## v1.7
-
-- refactoring version comparison logic.
-
-## v1.6
-
-- Adding in changelog url
-
-## v1.5
-
-- use the update path if you get a code 409
-- Excluding Policy Sync from publish
-- Updated policy template to not publish Policy Template Synchronization template
-
-## v1.4
-
-- Updating policy sync to use s3
-- Updating Policy Template Name
-
-## v1.3
-
-- Update email subject with account name and ID, and change actions and/or resolution name to be more descriptive. Issues #75 & #83
-
-## v1.2
-
-- Updating input parameter name for email
-
-## v1.1
-
-- Making email required.
-
-## v1.0
-
-- Initial Release
+# CHANGELOG
+
+## 1.11
+
+- Adding feature branch
```

Comment:
This change is probably more about adding support for publishing the the EU then adding feature branches.  Can we make this change?